src/actions/GraphDataActions.ts

```diff
@@ -60,7 +60,9 @@ const decorateGraphData = (graphData: any) => {
       graphData.nodes = graphData.nodes.map(node => {
         const decoratedNode = { ...node };
         if (node.data.traffic) {
-          node.data.traffic.map(protocol => {
+          const traffic = node.data.traffic;
```

Comment:
I moved all of this to a selector in a previous PR. The goal was to keep the same response from the server in our store.

See: https://github.com/kiali/kiali-ui/blob/master/src/store/Selectors/GraphData.ts#L10

Reply:
OK, I rebased and fixed things up.  Thanks.